src/cloud/components/onboarding/useGetUserStatus.ts

```diff
@@ -0,0 +1,110 @@
+import {useEffect, useState, useCallback, useMemo} from 'react'
+import {getOrgsUsage} from 'src/client'
+import {fromFlux} from '@influxdata/giraffe'
+import {usageStatsCsv} from 'src/shared/utils/mocks/usagestats'
+import {useSelector} from 'react-redux'
+import {getOrg} from 'src/organizations/selectors'
+import {event} from 'src/cloud/utils/reporting'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+
+export enum USER_PILOT_USER_STATUS {
+  NEW_USER = 'NEW_USER',
+  DELETED_ALL_DATA = 'DELETED_ALL_DATA',
+  NON_WRITING_USER = 'NON_WRITING_USER',
+  ACTIVE_USER = 'ACTIVE_USER',
+  ACTIVE_WRITING_USER = 'ACTIVE_WRITING_USER',
+  WRITING_NOT_READING_USER = 'WRITING_NOT_READING_USER',
+}
+export const getUserStatus = (table: any): USER_PILOT_USER_STATUS[] => {
+  const dataStates: USER_PILOT_USER_STATUS[] = []
+
+  const measurement = table.getColumn('_measurement') ?? []
+
+  const storageBytes = measurement.find((v, i) => {
+    return (
+      measurement[i] === 'storage_usage_bucket_bytes' &&
+      table.getColumn('_value')[i] > 0
```

Comment:
Should we specify a type here? What if this returns a string?

Reply:
ah yeah, great point